corehq/apps/app_manager/app_schemas/app_case_metadata.py

```diff
@@ -218,8 +218,8 @@ def add_form_contributions(self):
 
     def _add_load_update_actions(self):
         for action in self.form.actions.load_update_cases:
-            for name, question_path in action.case_properties.items():
-                self._add_property_save(action.case_type, name, question_path)
+            for name, conditional_update in action.case_properties.items():
+                self._add_property_save(action.case_type, name, conditional_update.question_path)
```

Comment:
Double-check me here:  Because we have not performed a full migration of the apps to the new model, I was worried this would break if the app wasn't wrapped before the user loaded the app summary page--but that shouldn't be possible because I believe the app is always wrapped upon opening app manager, and the summary page is inside app manager.

Reply:
That's correct.